src/helpers/dotNotation.js

```diff
@@ -0,0 +1,19 @@
+function dotNotation(number) {
+    const num = number.toString()
+    let string = num.split('');
+
+    string.reverse();
+    let test = string;
+    for (let i = 0; i < test.length; i++) {
+        if (i === 3) {
+            test.splice(3, 0, '.');
+        }
+        if (i === 7) {
+            test.splice(7, 0, '.');
+        }
+        test.concat(test[i])
```

Comment:
Nu werkt het niet als je 1000000000 hebt. Als je een wat robuustere oplossing wilt hebben, dan zul je moeten kijken of je iets kan met de [Remainder operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Remainder)